validator/client/validator-client-factory/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@prysm//tools/go:def.bzl", "go_library")
+
+config_setting(
+    name = "use_beacon_api",
+    values = {"define": "use_beacon_api=1"},
+)
```

Comment:
Thanks for the comment! I fixed it.

Reply:
In my opinion you still use `--define` that way. Take a look at https://docs.bazel.build/versions/0.29.1/skylark/config.html

Reply:
Ahh, got it. I misunderstood the intend of the deprecation warning. I have now added a build setting that can be called with `--use_beacon_api` instead of `--define=use_beacon_api=1`.